tests/functional/shrink_osd_legacy/hosts

```diff
@@ -0,0 +1,10 @@
+[mons]
+mon0 monitor_address=192.168.51.10
+
+[mgrs]
+mon0
+
+[osds]
+osd0 osd_scenario=non-collocated devices="['/dev/sda', '/dev/sdb']" dedicated_devices="['/dev/sdc', '/dev/sdc']" osd_objectstore=bluestore dmcrypt=true
```

Comment:
not sure to understand why testinfra is failing on osd0. The only difference with the other nodes is that there's only 2 OSDs vs 3 but conftest.py is supposed to get the devices length so it shouldn't be a problem.